src/index.js

```diff
@@ -1,10 +1,16 @@
 import './style.css';
 import getAllData from './getDataFood.js';
 
+// eslint-disable-next-line operator-linebreak
+const [liBeefElement, liPastaElement, liSeafoodElement] =
+  document.querySelectorAll('li');
+let typeFoodSelected = liBeefElement;
```

Comment:
I recommend here just to give ids to the links and call them as elements like this

  ```
  const beefButton = document.getElementById('beef');
  const pastaButton = document.getElementById('pasta');
  const seafoodButton = document.getElementById('seafood');

// event listeners
beefButton.addEventListener('click', () => getAllData(allBeefURL));
pastaButton.addEventListener('click', () => getAllData(allPastaURL));
seafoodButton.addEventListener('click', () => getAllData(allSeafoodURL));
  ```

and whit the counter that you can get from the getAllData function you can just append it when you click the link